index.js

```diff
@@ -57,6 +57,63 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "createPoll",
+    (msg, args) => {
+
+        const emojiSet = args[0];
+        let pollTitle = [];
+        let optionSet = [];
+        let optionBuilder = [];
+        let argCount = 0;
+
+        args.shift(); // Removes the first option that determins what emojis to use
+        args.push('99'); // Completely arbitrary number chosen to indicate the end of all the options. This is a bad idea and I need to figure out a different way to break up the options later.
+
+        for (let arg of args){
+            argCount++
+            if (arg === "Description:" ){
+                break;
+            }
+            pollTitle.push(arg);
+        }
+
+        args.splice(0, (argCount)); // Removes the Title
+
+        args.forEach( arg => { // Handles formating the discription
+            if ((Number(arg)) && (optionBuilder.length > 1) ) {
+                optionSet.push(optionBuilder.join(' '), '\n');
+                optionBuilder = [];
+            } 
+            else if (!(Number(arg))){ // This only adds the text without numbers
+                optionBuilder.push(arg);
+            };
```

Comment:
What if neither of these are true? Is that an error case? It might be good to at least document it, if not handle it.